src/Components/Notifications/NotificationItem.tsx

```diff
@@ -15,7 +15,6 @@ const UNREAD_INDICATOR_SIZE = 8
 
 const NotificationItem: React.FC<NotificationItemProps> = ({ item }) => {
   const artworks = extractNodes(item.artworksConnection)
-  const remainingArtworksCount = (item.artworksConnection?.totalCount ?? 0) - 4
```

Comment:
**suggestion**: I think you also need to remove [totalCount](https://github.com/artsy/force/blob/4740ec7917ae209321fd275ba892a890e215ecdc/src/Components/Notifications/NotificationItem.tsx#L97) from the query, since this data is not used

Reply:
thank you @dimatretyak, just did it!